uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/quotes/BookingQuotesView.kt

```diff
@@ -40,12 +38,18 @@ class BookingQuotesView @JvmOverloads constructor(context: Context,
         TextViewCompat.setTextAppearance(categoryText, detailsTextStyle)
     }
 
-    fun bindViews(url: String?, quoteName: String, category: String) {
+    fun bindViews(url: String?, quoteName: String, category: String, cancellationMinutes: Int?) {
         url?.let { loadImage(it) }
         quoteNameText.text = quoteName
         categoryText.text = String.format("%s%s",
                                           category.substring(0, 1).toUpperCase(),
                                           category.substring(1))
+        if (cancellationMinutes != null && cancellationMinutes > 0) {
```

Comment:
Can we move this to the presenter so we can unit test the logic? Seems overkill but I think at some point there might be additional logic required for the View.

Reply:
We have similar logic here too 
https://github.com/karhoo/karhoo-android-ui-sdk/pull/107/files#diff-421ae67ad42762bca6c3fb0f8a189b0444585c7e1cd94d719c0eb1bde20358cfR179

Perhaps as Jo suggested we can move the logic in Presenter and also share it with other places we need this logic ( very simple condition but worthwhile having one single source of truth)

Reply:
the BookingRequestView has reference to context, tou can get the `context.getString(R.string.uisdk_quote_cancellation_minutes)` and then calculate the correct value for `bookingQuoteCancellationText`  ... and then pass the value to 
`bindViews(url: String?, quoteName: String, category: String, cancellationSla: String?)` if the value is null, you can hide the view